Godeps/_workspace/src/bazil.org/fuse/syscallx/syscallx_std.go

```diff
@@ -3,24 +3,24 @@
 package syscallx
 
 // This file just contains wrappers for platforms that already have
-// the right stuff in stdlib.
+// the right stuff in golang.org/x/sys/unix.
 
 import (
-	"syscall"
+	"golang.org/x/sys/unix"
```

Comment:
no idea why 'godep save' didn't pick up this one... 

cc @jbenet - wait with merge until this is fixed